dockstore-webservice/src/main/java/io/openapi/api/impl/ToolsApiServiceImpl.java

```diff
@@ -292,20 +292,16 @@ public Response toolsGet(String id, String alias, String toolClass, String descr
                 all.addAll(toolDAO.findAllPublished());
```

Comment:
This would be a little bit hackish, but if the descriptorType is NextFlow or Galaxy, we could skip this step. You could hard-code, with a TODO, although ideally DescriptorLanguage would have something that indicates whether the language supports tools (not sure if that has API implications though).